src/vivarium_gates_nutrition_optimization/data/loader.py

```diff
@@ -377,7 +377,7 @@ def get_moderate_hemorrhage_probability(key: str, location: str) -> pd.DataFrame
     # random seed
     rng = np.random.default_rng(get_hash(f"hemorrhage_severity"))
     moderate_hemorrhage_probability = pd.DataFrame(
-        [dist.rvs(size=1000, random_state=rng)],
+        [dist.rvs(size=500, random_state=rng)],
```

Comment:
Just making sure this also gets updated to vi_globals.DRAW_COLUMNS

Reply:
Yep!